addons/website_sale/views/templates.xml

```diff
@@ -614,27 +614,39 @@
                                 </li>
                             </ol>
                         </div>
-                        <div class="col-md-8">
-                            <div class="form-inline justify-content-end">
+                        <div class="col-lg-6">
+                            <div class="d-sm-flex mb-2">
                                 <t t-call="website_sale.search">
                                     <t t-set="search" t-value="False"/>
+                                    <t t-set="_classes" t-value="'mb-2 mr-sm-2'"/>
                                 </t>
                                 <t t-call="website_sale.pricelist_list">
-                                    <t t-set="_classes" t-valuef="ml-2"/>
+                                    <t t-set="_classes" t-valuef="mb-2 float-right"/>
                                 </t>
                             </div>
                         </div>
                     </div>
                     <div class="row">
-                        <div class="col-md-6 col-xl-8">
+                        <div class="col-md-6 mt-md-4">
                             <t t-call="website_sale.shop_product_carousel"/>
                         </div>
-                        <div class="col-md-6 col-xl-4" id="product_details">
+                        <div class="col-md-6 mt-md-4" id="product_details">
                             <h1 itemprop="name" t-field="product.name">Product Name</h1>
                             <span itemprop="url" style="display:none;" t-esc="product.website_url"/>
+                            <t t-if="is_view_active('website_sale.product_comment')">
+                                <a href="#o_product_page_reviews" class="o_product_page_reviews_link text-decoration-none">
+                                    <t t-call="portal_rating.rating_widget_stars_static">
+                                        <t t-set="rating_avg" t-value="product.rating_avg"/>
+                                        <t t-set="trans_text">%s reviews</t>
+                                        <t t-set="rating_count" t-value="trans_text % product.rating_count"/>
```

Comment:
This is a trick from BSO, waiting for a better solution to translate text in t-value attribute